source/tutorials.txt

```diff
@@ -11,5 +11,6 @@ Tutorials
    /tutorials/version-bumping
    /tutorials/generating-a-browser-list
    /error-kb
+   /tutorials/install 
    /tutorials/repo
```

Comment:
/tutorials/repo.txt will need to have the following changes:

- Capitalize bulleted lists (What You'll Need, Check Your Environment, and See Also)